templates/status.html

```diff
@@ -63,14 +63,16 @@
         function main ()  {
             // list of supported websites
             var sites = "{{sites}}".split(" ");
+            var sitesCss = JSON.parse({{ css_rules|tojson|safe }});
 
             // create cards dynamically for each of the websites
             sites.forEach(website => {
+              var iconElement = constructIconElement(website, sitesCss);
               $(".helper").append(`
                 <div class="card">
                     <p>
                         <div class="tooltip">
-                            <i class="fa fa-${website}"></i>
+                            <${iconElement.htmlElement} ${iconElement.htmlAttribute}="${iconElement.attributeValue}"></${iconElement.htmlElement}>
```

Comment:
This line is a bit too long, can you put `</${iconElement.htmlElement}>` to the next line?